workshop/serializers.py

```diff
@@ -637,7 +638,7 @@ class Meta:
             'id', 'title', 'description', 'club', 'entity', 'date', 'time', 'location',
             'latitude', 'longitude', 'audience', 'resources', 'contacts', 'image_url',
             'is_interested', 'interested_users', 'is_workshop_contact', 'is_por_holder', 'tags',
-            'link')
+            'link', 'is_workshop')
```

Comment:
I think this serializer is also used in updating workshops. So, `is_workshop` can be added in the `read_only_fields`